files/var/kernel/README.md

```diff
@@ -1,22 +1,25 @@
 Building the kernel
 ===================
 
+Best to do this on a fresh VM in a cloud, running the desired distribution.
+
+* `rev=5.4.0-1024-aws`
 * Enable source repositories in `/etc/apt/sources.list`
 ```
-  deb-src http://archive.ubuntu.com/ubuntu bionic main
-  deb-src http://archive.ubuntu.com/ubuntu bionic-updates main
+deb-src http://archive.ubuntu.com/ubuntu bionic main
+deb-src http://archive.ubuntu.com/ubuntu bionic-updates main
 ```
-* `sudo apt-get build-dep linux linux-image-$(uname -r)`
-* `sudo apt-get install libncurses-dev flex bison openssl libssl-dev dkms libelf-dev libudev-dev libpci-dev libiberty-dev autoconf`
-* `apt-get source linux-image-$(uname -r)`
-* `cd linux-image*`
+* `sudo apt-get build-dep linux linux-image-$rev`
+* `sudo apt-get install libncurses-dev flex bison openssl libssl-dev dkms libelf-dev libudev-dev libpci-dev libiberty-dev autoconf fakeroot gcc-aarch64-linux-gnu`
```

Comment:
What is the cross compiler gcc toolchain needed for? (I see only amd64 debian packages in /files/var/kernel)